### PR TITLE
Add database schema

### PR DESCRIPTION
**Pull request by kaniak274**

### CODE REVIEW

File: books/__init__.py [hunk 1]
```diff
@@ -10,9 +10,11 @@ def create_app():
     app.config.from_mapping(SECRET_KEY='books')
 
     from . import database
-    from .schema import schema
 
-    app.add_url_rule('/books', view_func=GraphQLView.as_view('graphql', schema=schema, graphiql=True))
+    ### TODO
+    # from .schema import schema
```
Comment (KamilSupera) [inline books/__init__.py:15]: Why u comment on these lines?

File: books/database.py [hunk 2]
```diff
@@ -1,7 +1,47 @@
 import os
 
-from pymongo import MongoClient
+from sqlalchemy import create_engine, Column, Date, ForeignKey, Integer, MetaData, String, Table, Text
```
Comment (KamilSupera) [inline books/database.py]: I think this line is to long

File: books/__init__.py (hunk 1 above)
Comment (kaniak274) [reply books/__init__.py:15]: Because I want to change it will crush now because no queries are specified

File: books/database.py (hunk 2 above)
Comment (kaniak274) [reply books/database.py]: Yeah you're right

**Review by KamilSupera — APPROVED** [review]
Look at conflict :)

**Pull request merged**